tools/profileBuilder/README.md

```diff
@@ -0,0 +1,48 @@
+# Profile Builder
+
+## Installation
+
+*Note:* These installation notes assume that you have [Go 1.9](https://blog.golang.org/go1.9) or higher, [Glide](http://github.com/Masterminds/glide), and [Git](https://git-scm.com/) installed.
+
+The simplest version of installation is simple, just run the follwoing command:
+
+``` bash
+go get -u github.com/Azure/azure-sdk-for-go/tools/profileBuilder
+```
+
+If that causes you trouble, run the following commands:
+
+``` bash
+# bash
+go get -d github.com/Azure/azure-sdk-for-go/tools/profileBuilder
+cd $GOPATH/src/github.com/Azure/azure-sdk-for-go/tools/profileBuilder
+glide install
+go install
+```
+
+``` PowerShell
+# PowerShell
+go get -d github.com/Azure/azure-sdk-for-go/tools/profileBuilder
+cd $env:GOPATH\src\github.com\Azure\azure-sdk-for-go\tools\profileBuilder
+glide install
+go install
+```
+Taking things a step further, if you'd like the profileBuilder to stamp the version of itself in the code it generates, you can install as below:
+
+``` bash
+# bash
+go get -d github.com/Azure/azure-sdk-for-go/tools/profileBuilder
+cd $GOPATH/src/github.com/Azure/azure-sdk-for-go/tools/profileBuilder
+glide install
+export currentCommit=$(git rev-parse HEAD)
```

Comment:
>git rev-parse HEAD [](start = 23, length = 18)

Could this be executed inside the profile builder, and just have a bool flag for including or not the commit?

Reply:
Actually, it cannot be done this way :) That would result in the profileBuilder always telling you it was the most up-to-date version even if it wasn't. We really need to set this variable at build time the way I've done it.